### ISSUE TITLE
TestRailNotifier.java TestRailException is never thrown

### ISSUE THREAD

**Issue by VonC** [jenkinsci/testrail-plugin#8]
A simple `mvn clean install` fails with the current https://github.com/jenkinsci/testrail-plugin/commit/dc688eb7832a7fd58994f1e8c00b14c85eb710b9 last commit from April 29th.

That last commit introduced a `catch TestRailException`  in `doFillTestrailSuiteItems()` : https://github.com/jenkinsci/testrail-plugin/commit/dc688eb7832a7fd58994f1e8c00b14c85eb710b9#diff-9fe7050b076aaf5d5693ff2c65860e26R290 

That is enough to make the compilation fail:

````
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /C:/Users/vonc/prog/git/testrail-plugin/src/main/java/org/jenkinsci/plugins/testrail/TestRailNotifier.java:[290,15] exception org.jenkinsci.plugins.testrail.TestRailObjects.TestRailException is never thrown in body of corresponding try statement
````

Without that catch, the project does compile.  
A PR is coming.

**Comment by xirsoi** [comment]
That's bizarre, I know I ran a build before committing that change... Thank you for the ticket and the upcoming PR!

**Comment by xirsoi** [comment]
Fixed with PR #9.
